### ISSUE TITLE
new check: users with no password

### ISSUE THREAD

**Issue by trel** [irods/irods_client_zone_management_tool#114]
- except 'anonymous'
- except 'groups'

**Comment by ganning127** [comment]
I don't think it is possible to do this right now on ZMT. When looking at the genquery columns, there is no column that represents a user's password, or whether or not their password is empty. 

I think an addition on the server side would need t be made, to add a flag that says whether or not this users' password is empty and expose that as a column on genquery.

If there's anything I missed, please let me know!
